### ISSUE TITLE
Password Change error in NagVis

### ISSUE THREAD

**Issue by gabortakacs78** [NagVis/nagvis#215]
Users get the following message when they try to initiate Password Change from menu:
ERROR: You are not permitted to access this page (ChangePassword/view/).

Nagvis version: v1.9.10 & v1.9.14

More details:
https://monitoring-portal.org/t/change-nagvis-user-password-not-possible/6371/4

Could it be it is really not working since v1.9.x?
Revert to old source code "server/core/classes/CoreModChangePassword.php" solved our issue temporarily.
Please check if problem could be really on developer side!!!

Thanks in advance!

**Comment by vpithart** [comment]
Hey @gabortakacs78, it definitely is an error. There's a quick fix you can do on your server before it get fixed in nagvis code. Applicable to v1.9.14+.

## quick fix

Locate the `auth.db` file within your installation, ie.
```sh
ls -la  /usr/local/nagvis/etc/auth.db
-rw-r--r-- 1 www-data www-data 51200 jan  1 11:11 /usr/local/nagvis/etc/auth.db
```

Open it with `sqlite3` utility
```sh
sqlite3  /usr/local/nagvis/etc/auth.db
```

And change the problematic record:
```sql
sqlite> update perms set act='*' where mod='ChangePassword' and act='change';
```

**Comment by gabortakacs78** [comment]
Hi,

I've tested, and it works well now.
So I reverted back my changes in "server/core/classes/CoreModChangePassword.php" file, and update sqlite DB as you suggested.

Thanks a lot for your help!

**Comment by LarsMichelsen** [comment]
Solved by #217
